shared/actions/chat2/index.js

```diff
@@ -633,6 +639,12 @@ const loadMoreMessages = (
   } else if (action.type === Chat2Gen.selectConversation) {
     key = action.payload.conversationIDKey
     reason = action.payload.reason || 'selected'
+  } else if (action.type === Chat2Gen.metasReceived) {
+    if (!action.payload.clearExistingMessages) {
+      logger.info("Load thread bail: metas received but haven't cleared the existing messages")
```

Comment:
i don't think we need this log really, we do receive metasReceived a bunch and its only this circumstance which we care about so we don't need to log ignored side effect triggers